db/migrate/20180109143959_add_remember_token_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddRememberTokenToUsers < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :remember_token, :string, null: true
```

Comment:
I don't see what is using this column?

Reply:
this allows remember_me for models without password. Sadly it was the cleanest solution possible.